protocol/messenger_communities.go

```diff
@@ -1098,6 +1130,226 @@ func (m *Messenger) handleSyncCommunity(messageState *ReceivedMessageState, sync
 	return nil
 }
 
+func (m *Messenger) InitHistoryArchiveTasks(communities []*communities.Community) {
+
+	dispatchMagnetlink := func(communityID types.HexBytes) {
+		err := m.dispatchMagnetlinkMessage(communityID)
+		if err != nil {
+			m.logger.Debug("failed to dispatch magnetlink message", zap.Error(err))
+		}
+	}
+
+	for _, c := range communities {
+
+		if c.Joined() {
+			settings, err := m.communitiesManager.GetCommunitySettingsByID(c.ID())
+			if err != nil {
+				log.Println("failed to get community settings", err)
+				m.logger.Debug("failed to get community settings", zap.Error(err))
+				continue
+			}
+			if !settings.HistoryArchiveSupportEnabled {
+				continue
+			}
+
+			filters, err := m.communitiesManager.GetCommunityChatsFilters(c.ID())
+			if err != nil {
+				log.Println("failed to get community chats filters", err)
+				m.logger.Debug("failed to get community chats filters", zap.Error(err))
+				continue
+			}
+
+			if len(filters) == 0 {
+				log.Println("no filters or chats for this community", c.IDString())
+				m.logger.Debug("no filters or chats for this community")
+				continue
+			}
```

Comment:
Actually, this is an issue... When we create a community it won't have any filters because it won't have any chats yet. This means, when `InitHistoryArchiveTasks` is called, it won't schedule the interval. We should schedule it here.

Reply:
This has been addressed by now..